nemo_skills/inference/launch_chat_interface.py

```diff
@@ -22,14 +22,31 @@ def parse_args() -> argparse.Namespace:
         default="localhost",
         help="Hostname where both the model server and the execution sandbox are running.",
     )
+    parser.add_argument(
+        "--ssh_server",
```

Comment:
are these used anywhere? Best to also accept from our standard env vars for this